<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrackWidget</class>
 <widget class="QWidget" name="TrackWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Track Filter Options</string>
  </property>
  <property name="whatsThis">
   <string>This filter performs various operations on track data. </string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="text">
        <string>Track Filters</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="titleCheck">
       <property name="toolTip">
        <string>Basic title for track</string>
       </property>
       <property name="whatsThis">
        <string>Basic title for new track(s). 
This option specifies a title for tracks generated by the track filter. By default, the title of the new track is composed of the start time of the track appended to this value. 
If this value contains a percent (%) character, it is treated as a format string for the POSIX strftime function, allowing custom time-based track names. </string>
       </property>
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="4">
      <widget class="QLineEdit" name="titleText">
       <property name="toolTip">
        <string>the title of the new track is composed of the start time of the track appended to this value. </string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="moveCheck">
       <property name="toolTip">
        <string>Correct track point timestamps by specified amount</string>
       </property>
       <property name="whatsThis">
        <string>Correct trackpoint timestamps by a delta. 
This option changes the time of all trackpoints. This might be useful if your track must be moved by one or more hours because of an incorrect time zone. </string>
       </property>
       <property name="text">
        <string>Move</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QSpinBox" name="weeksSpin">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="minimum">
          <number>-10000</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="weeksLabel">
         <property name="text">
          <string>weeks</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QSpinBox" name="daysSpin">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="minimum">
          <number>-100000</number>
         </property>
         <property name="maximum">
          <number>100000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="daysLabel">
         <property name="text">
          <string>days</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="3">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QSpinBox" name="hoursSpin">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="minimum">
          <number>-100000</number>
         </property>
         <property name="maximum">
          <number>100000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="hoursLabel">
         <property name="text">
          <string>hours</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="4">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSpinBox" name="minsSpin">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="minimum">
          <number>-100000</number>
         </property>
         <property name="maximum">
          <number>100000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="minsLabel">
         <property name="text">
          <string>mins</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="5">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="secsSpin">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="minimum">
          <number>-100000</number>
         </property>
         <property name="maximum">
          <number>100000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="secsLabel">
         <property name="text">
          <string>secs</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="startCheck">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Use track pts. after this time. </string>
       </property>
       <property name="whatsThis">
        <string>Use only track points after this timestamp.

This option is used along with the stop to discard trackpoints that were recorded outside of a specific period of time. This option specifies the beginning of the time period. </string>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QDateTimeEdit" name="startEdit">
       <property name="toolTip">
        <string>Use track pts. after this time. </string>
       </property>
      </widget>
     </item>
     <item row="3" column="3" rowspan="2">
      <widget class="QCheckBox" name="TZCheck">
       <property name="toolTip">
        <string>If checked, time specified here is based on this computer's current time zone. </string>
       </property>
       <property name="whatsThis">
        <string>If checked, the times specified here are based on the local computer's time zone.  Otherwise it is UTC.</string>
       </property>
       <property name="text">
        <string>Local Time</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="stopCheck">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Use track pts before this time. </string>
       </property>
       <property name="whatsThis">
        <string> Use only track points before this timestamp.

This option is used in conjunction with the start option to discard all trackpoints outside of a given period of time. This option defines the end of the time period. </string>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QDateTimeEdit" name="stopEdit">
       <property name="toolTip">
        <string>Use track pts before this time. </string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="packCheck">
       <property name="toolTip">
        <string>Pack all tracks into one. </string>
       </property>
       <property name="whatsThis">
        <string>Pack all tracks into one.

This option causes all tracks to be appended to one another to form a single track. This option does not work if any two tracks overlap in time; in that case, consider using the merge option.

This option is most useful for rejoining tracks that might have been interrupted by an equipment malfunction or an overnight stop. </string>
       </property>
       <property name="text">
        <string>Pack</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="mergeCheck">
       <property name="toolTip">
        <string>Merge multiple tracks for the same way.</string>
       </property>
       <property name="whatsThis">
        <string> Merge multiple tracks for the same way.

This option puts all track points from all tracks into a single track and sorts them by time stamp. Points with identical time stamps will be dropped. </string>
       </property>
       <property name="text">
        <string>Merge</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QRadioButton" name="splitDateCheck">
       <property name="text">
        <string>Split by Date</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QRadioButton" name="splitTimeCheck">
       <property name="text">
        <string>Split by Time</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QSpinBox" name="splitTimeSpin">
         <property name="toolTip">
          <string>If nonzero, the track will be split if the time between two points is greater than this parameter.   If zero, the track will be split by date. </string>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="splitTimeCombo">
         <item>
          <property name="text">
           <string>mins</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>hrs</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>days</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="4">
      <widget class="QRadioButton" name="splitDistanceCheck">
       <property name="text">
        <string>Split by Dist.</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="5">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QSpinBox" name="splitDistSpin">
         <property name="toolTip">
          <string>If nonzero, the input track will be split into several tracks if the distance between successive track points is greater than the distance given as a parameter. </string>
         </property>
         <property name="maximum">
          <number>5280</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="splitDistCombo">
         <item>
          <property name="text">
           <string>ft</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>m</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>km</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>mi</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QCheckBox" name="GPSFixesCheck">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>GPS Fixes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="GPSFixesCombo">
         <item>
          <property name="text">
           <string>none</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>pps</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>dgps</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3d</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2d</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>2</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="7" column="4">
      <widget class="QCheckBox" name="speedCheck">
       <property name="toolTip">
        <string>Synthesize speed. </string>
       </property>
       <property name="whatsThis">
        <string>Synthesize speed.

This option computes a value for the GPS speed at each trackpoint. This is most useful with trackpoints from formats that don't support speed information or for trackpoints synthesized by the interpolate filter. The speed at each trackpoint is the average speed from the previous trackpoint (distance divided by time). The first trackpoint in each track is assigned a speed of &quot;unknown.&quot; </string>
       </property>
       <property name="text">
        <string>Speed</string>
       </property>
      </widget>
     </item>
     <item row="7" column="5">
      <widget class="QCheckBox" name="courseCheck">
       <property name="toolTip">
        <string>Synthesize course.</string>
       </property>
       <property name="whatsThis">
        <string>Synthesize course.

This option computes (or recomputes) a value for the GPS heading at each trackpoint. This is most useful with trackpoints from formats that don't support heading information or for trackpoints synthesized by the interpolate filter. The heading at each trackpoint is simply the course from the previous trackpoint in the track. The first trackpoint in each track is arbitrarily assigned a heading of 0 degrees. </string>
       </property>
       <property name="text">
        <string>Course</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>1</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
